### ISSUE TITLE
Scale Sensitive Loss的系数计算

### ISSUE THREAD

**Issue by tuoyuantenghai** [Lliu666/MSHNet#21]
您好，按照您文章里提供的w计算公式，以及源码中loss.py关于w的计算
    def forward(self, pred_log, target,warm_epoch, epoch, with_shape=True):
        pred = torch.sigmoid(pred_log)
        smooth = 0.0

        intersection = pred * target

        intersection_sum = torch.sum(intersection, dim=(1,2,3))
        pred_sum = torch.sum(pred, dim=(1,2,3))
        target_sum = torch.sum(target, dim=(1,2,3))
        
        dis = torch.pow((pred_sum-target_sum)/2, 2)
        
        
        alpha = (torch.min(pred_sum, target_sum) + dis + smooth) / (torch.max(pred_sum, target_sum) + dis + smooth) 
这无法和您的图三左图印证（如Ap=100，Agt=0，w=2500/2600=0.9615而非图中画的0，并且当Ap=100，Agt=90/60时，w均等于0.92），左图貌似和w=min（Ap，Agt）/max（Ap，Agt）的计算结果相吻合
望能抽空解答疑问，感谢
